### PR TITLE
chore: cds 8 upgrade

### PR DESCRIPTION
**Pull request by johannes-vogel**

### CODE REVIEW

**Comment by stewsk** [comment]
Errors in Java test
```
com.sap.cds.CdsException: CDS Compiler version 5 is not supported. Please use cds-dk 7 w/ cds-compiler 4
```
@agoerler : do we have to simultaneously upgrade also Java?

**Comment by agoerler** [comment]
> @agoerler : do we have to simultaneously upgrade Java RT?

Sure. But it has not been released yet.

File: package.json [hunk 1]
```diff
@@ -112,10 +109,6 @@
           }
         ]
       }
-    },
-    "sql": {
-      "transitive_localized_views" : false,
-      "native_hana_associations" : false
```
Comment (agoerler) [inline package.json:118]: We need to keep this config as I think the default is still `true`.

File: package.json [hunk 2]
```diff
@@ -112,10 +109,6 @@
           }
         ]
       }
-    },
-    "sql": {
-      "transitive_localized_views" : false,
```
Comment (agoerler) [inline package.json:117]: We need to check if this config fixes the issue on H2.

Comment (agoerler) [reply package.json:117]: Reintroducing this config did not help for H2.

File: package.json (hunk 1 above)
Comment (johannes-vogel) [reply package.json:118]: true, I'll revert.

File: package.json (hunk 2 above)
Comment (johannes-vogel) [reply package.json:117]: I think we should look at this again. I don't expect that this should be part of the default configuration.

Comment (agoerler) [reply package.json:117]: The default ("transitive_localized_views" : false) to be working. We don't need to re-introduce the config.

**Pull request merged**